.tekton/keystone-operator-pull-request.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/raukadah/keystone-operator-konflux-integration?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    # when adding filter for directory use "directory/***".pathChanged()
+    # when adding filter for git submodule use "submodule".pathChanged()
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch == "konflux_main" &&
+      (".tekton/keystone-operator-pull-request.yaml".pathChanged())
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: test-component
+    appstudio.openshift.io/component: test-component
+    pipelines.appstudio.openshift.io/type: build
+  name: gatekeeper-on-pull-request
+  namespace: user-ns2
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/raukadahkonfluxci/user-ns2/test-component/keystone-operator:on-pr-{{revision}}
+  - name: image-expires-after
+    value: 5d
+  - name: dockerfile
+    value: Dockerfile
+  - name: path-context
+    value: .
+  - name: build-platforms
+    value:
+      - localhost
+      - linux/arm64
+  #     - linux/ppc64le
+  #     - linux/s390x
+  pipelineRef:
+    name: build-pipeline
```

Comment:
It should be `single-arch-build-pipeline`. As .tekton/single-arch-build-pipeline.yaml has 
```
metadata:
  name: single-arch-build-pipeline
```

Reply:
Yes, it got fixed